index.bs

```diff
@@ -101,6 +101,28 @@ See also
 
 * [[DESIGN-PRINCIPLES#priority-of-constituencies]]
 
+When answering this question, please consider each of these four possible
+areas of information disclosure / sharing.
+
+For the below sub-questions, please take the term
+*potentially identifying information* to mean information
+that describes the browser user, distinct from other users of the same browser.
+Examples of such *potentially identifying information* includes information
+about the browser user's environment (e.g., operating system configuration,
+browser configuration, hardware capabilities), and the user's prior activities
+and interests (e.g., browsing history, purchasing preferences, personal
+characteristics).
+
+1.  What information does your spec expose to the **first party** that
+    the **first party** cannot currently easily determine.
+2.  What information does your spec expose to **third parties** that
+    the **third parties** cannot currently easily determine.
+3.  What *potentially sensitive information* does your spec expose to the
```

Comment:
Should this be `*potentially identifying information*`?

Reply:
Good catch, fixed!